dockers/docker-auditd/config_checker.py

```diff
@@ -46,18 +47,23 @@ def run_command(cmd):
     return p.returncode, error
 
 
-def get_hwsku():
-    hwsku_cmd = "sonic-cfggen -d -v DEVICE_METADATA.localhost.hwsku"
-    rc, out = run_command(hwsku_cmd)
+def get_arch():
+    arch_cmd = NSENTER_CMD + "uname -m"
```

Comment:
>uname [](http://example.com/codeflow?start=30&length=5)

Sorry for misleading, not sure if vendor hardware minor interaction will keep uname output unchanged. Then using sonic-cfggen hwsku may be more stable solution.